src/Disks/IO/ThreadPoolRemoteFSReader.cpp

```diff
@@ -121,6 +124,7 @@ IAsynchronousReader::Result ThreadPoolRemoteFSReader::execute(Request request, b
 
     auto * fd = assert_cast<RemoteFSFileDescriptor *>(request.descriptor.get());
     auto & reader = fd->getReader();
+    auto * cached_reader = typeid_cast<CachedInMemoryReadBufferFromFile *>(&reader);
```

Comment:
```suggestion
    auto * page_cache_reader = typeid_cast<CachedInMemoryReadBufferFromFile *>(&reader);
```
Technically `cached_reader` could also mean `CachedOnDiskReadBufferFromFile`, not only `CachedInMemoryReadBufferFromFile`, so `cached_reader` here is misleading.